packages/features/webhooks/components/WebhookTestDisclosure.tsx

```diff
@@ -34,12 +30,24 @@ export default function WebhookTestDisclosure() {
           {t("ping_test")}
         </Button>
       </div>
-      <div className="p-4">
-        {!mutation.data && <em>{t("no_data_yet")}</em>}
-        {mutation.status === "success" && (
-          <div className="overflow-x-auto  text-gray-900">{JSON.stringify(mutation.data, null, 4)}</div>
-        )}
+      <div className="space-y-0 rounded-md border-0 border-neutral-200 bg-white sm:mx-0 md:border">
```

Comment:
```suggestion
      <div className="space-y-0 rounded-md border-0 border-neutral-200 sm:mx-0 md:border">
```

Reply:
Sorry I didn't notice it. Thanks, @emrysal for pointing it out.